### PR TITLE
Update Tresorit in backup.yml

### PR DESCRIPTION
**Pull request by jamcat22**
They have 2FA now! https://support.tresorit.com/entries/104192996-How-to-set-up-two-step-verification

### CODE REVIEW

**Comment by jamcat22** [comment]
I really need to fix my fork. XD

File: _data/backup.yml [hunk 1]
```diff
@@ -142,10 +142,12 @@ websites:
 
   - name: Tresorit
     url: https://tresorit.com/
-    twitter: Tresorit
```
Comment (Walkman100) [inline _data/backup.yml:145]: ?

Comment (mxxcon) [inline _data/backup.yml:145]: @Walkman100 twitter info is mainly relevant for public shaming. if a site has 2fa, twitter info is not displayed, so not as critical to keep in yaml.

Comment (Walkman100) [inline _data/backup.yml:145]: Ah I see, I thought the Twitter handle was displayed whether or not TFA is supported, for some reason.

**Pull request merged**